src/edu/ucsb/nceas/metacat/restservice/v1/MNResourceHandler.java

```diff
@@ -1084,7 +1085,8 @@ private void getLog() throws InvalidToken, ServiceFailure, NotAuthorized, Invali
         }
 
         logMetacat.debug("calling getLogRecords");
-        Log log = MNodeService.getInstance(request).getLogRecords(session, fromDate, toDate, event, pidFilter, start, count);
+        org.dataone.service.types.v1.Log log = MNodeService.getInstance(request)
```

Comment:
I added new imported  class `org.apache.commons.logging.Log` since each ResourceHandler class has its own log object. So there are duplicate Log class and I have to use the full name here.